src/ucs/stats/serialization.c

```diff
@@ -307,14 +307,27 @@ ucs_stats_serialize_text_recurs(FILE *stream, ucs_stats_node_t *node,
                                 ucs_stats_children_sel_t sel, unsigned indent)
 {
     ucs_stats_node_t *child;
-    unsigned i;
+    unsigned i, j;
+    int found_group = 0;
+
+    for (j = 0; j < ucs_global_opts.stats_included_groups.count; ++j) {
```

Comment:
pls move ucp_str_array_search() into ucs config, make it accept pointer to ucs_config_names_array_t, and reuse it in this code